packages/legend-graph/src/graphManager/protocol/pure/v1/model/packageableElements/store/flatData/model/V1_FlatDataProperty.ts

```diff
@@ -19,13 +19,13 @@ import { type Hashable, hashArray } from '@finos/legend-shared';
 
 export class V1_FlatDataProperty implements Hashable {
   name!: string;
-  value!: boolean | string | number;
+  value!: (boolean | string | number)[];
```

Comment:
We can also create a type `V1_FlatDataPropertyValue = boolean | string | number` here